src/apps/Hosting/HostingFilesStore.js

```diff
@@ -66,6 +66,15 @@ export default Reflux.createStore({
     this.data.currentHostingId = hostingId;
   },
 
+  handleCloseOnUpload(event) {
+    event.preventDefault();
+    event.returnValue = 'Are you sure you want to close?';
```

Comment:
I meant not many people see custom text. Confirm dialog displays well, so on my mind it's good way.